### PR TITLE
RavenDB-13046 Null result for swap definition fixed

### PR DESCRIPTION
**Pull request by haludi**
https://issues.hibernatingrhinos.com/issue/RavenDB-13046

### CODE REVIEW

File: src/Sparrow/LowMemory/CheckPageFileOnHdd.cs [hunk 1]
```diff
@@ -397,9 +397,7 @@ public static string PosixIsSwappingOnHddInsteadOfSsd()
                         continue; // we do not check swap file, only partitions
 
                     // remove numbers at end of string (i.e.: /dev/sda5 ==> sda)
-                    var disk = swaps[i].DeviceName != null
-                        ? _regExRemoveNumbers.Replace(swaps[i].DeviceName, "").Replace("/dev/", "")
-                        : _regExRemoveNumbers.Replace("/dev/", "");
+                    var disk = _regExRemoveNumbers.Replace(swaps[i].DeviceName, "").Replace("/dev/", "");
```
Comment (aviviadi) [inline src/Sparrow/LowMemory/CheckPageFileOnHdd.cs:400]: Do we have here swaps[i].DeviceName null error handling?

File: test/FastTests/Sparrow/FileSystemUtilsTest.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            var swap = KernelVirtualFileSystemUtils.ReadSwapInformationFromSwapsFile();
+            Assert.All(swap, s => Assert.NotNull(s.DeviceName));
+        }
+        
+        [LinuxFact]
+        public void WriteAndReadPageUsingCryptoPager_WhenTwoSwapDefined_AllResultShouldBeNotNull()
+        {
+            var fileName = GetTempFileName();
+            
+            string[] lines =
```
Comment (aviviadi) [inline test/FastTests/Sparrow/FileSystemUtilsTest.cs:21]: Please add also more `lines` to test:
```
{
                "Filename				Type		Size	Used	Priority"
}
```
and
```
{
                "Filename				Type		Size	Used	Priority",
                ""
}

and 
```
{
                "Filename				Type		Size	Used	Priority"
                "/dev/sda                               partition	999420	56320	-2", 
}
```
and also with filename

File: src/Sparrow/LowMemory/CheckPageFileOnHdd.cs (hunk 1 above)
Comment (haludi) [reply src/Sparrow/LowMemory/CheckPageFileOnHdd.cs:400]: We can't get null at `DeviceName` as it implemented now and we shouldn't. 
I added a test to check it.
I think if we still want to check this it should be by `Debug.Assert`

**Review by aviviadi — APPROVED** [review]

File: src/Sparrow/LowMemory/CheckPageFileOnHdd.cs (hunk 1 above)
Comment (aviviadi) [reply src/Sparrow/LowMemory/CheckPageFileOnHdd.cs:400]: PR Approved

**Pull request merged**